tests/testthat/test-get_varcov.R

```diff
@@ -5,9 +5,12 @@ test_that("informative error in get_varcov.default", {
     requiet("lme4")
     mod <- lmer(mpg ~ hp + (1 | cyl), data = mtcars)
     # sandwich: not supported
-    expect_error(get_varcov(mod, vcov = "HC2"), regexp = "not be supported")
```

Comment:
@vincentarelbundock Not sure, but can it be that due to line breaks in the warning message, the regex might be different here? To be safe, I just removed the test against the regex.

Reply:
Oh right. That's totally possible.